examples/sha256sum/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+	"github.com/bitfield/script"
+	"log"
+	"os"
+	"strings"
+)
+
+func main() {
+	// Read the first 2 args, and calculate the checksum of the files.
+	hashFiles, err := script.Args().First(2).SHA256Sums().String()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// Compare the SHA256 checksum to check if files are identical.
+	checkSums := strings.Split(hashFiles,"\n")
```

Comment:
Seems to me we could use a `Slice()` sink, doesn't it? There are now several filters which produce a pipe full of newline-separated strings, and it would be convenient to get these right into a `[]string`.

Reply:
(Not asking you to implement that; this is #45)

Reply:
Yes, having a `Slice()` method could be awesome.